Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM python:3.11-slim-bullseye
 ENV FLASK_ENV=dev
-ENV GUNICORN_CMD_ARGS="--timeout 60 --workers 3"
+ENV GUNICORN_CMD_ARGS="--timeout 300 --workers 3"
```

Comment:
I made this change to fix an issue in test yesterday where the worker timed out during ingest. We should probably add a TODO though to check if this is a suitable long term solution to the problem